VERSIONING.md

```diff
@@ -12,76 +12,92 @@ Minor releases contain new smaller features. Minor releases are fully backward-c
 
 Patch releases are low risk, bug fix releases. No developer assistance is expected during update.
 
-# Branches
+# Branching
 
-## Working branches
+Currently, two popular development approaches can be considered for branching: Git flow and trunk-based development. Both have their own pros and cons.
 
-Working branches should be named according to the following pattern:
+In most cases we are going to use Git flow for our projects, but for the libraries, trunk-based development could be the best option to iterate quickly.
 
-{BaseBranch}-{Type}{Issue#}-{IssueShortTitle}
 
-Where:
-- {BaseBranch}: Name of the maintenance branch or master used to create the working branch
-- {Type}: Letter indicating the type of issue addressed on the branch
-    - F: New feature
-    - B: Bug
-- {Issue#}: GitHub issue number associated to the feature or defect to be addressed on the branch. That would allow better tracking between branches and change description.
-- {ShortTitle}: Short title of the change. Written using [UpperCamelCase](https://es.wikipedia.org/wiki/CamelCase)
+## Git FLow
 
-Examples
-- master-B12-DisableDragAndDrop
-- 4.1.x-F68-RoleDirective
+In the Git flow development model, one main development branch is considered.
+
+Feature branches are created from de development branch and once they are done, pull requests are created.
```

Comment:
Just a typo: from THE development branch

Reply:
OK